pcp/screens/filesys

```diff
@@ -0,0 +1,77 @@
+#
+# pcp-htop(1) configuration file - see pcp-htop(5)
+#
+
+[filesys]
+caption = Filesystem
+enabled = true
+
+avail.heading = AVAIL
+avail.caption = avail
+avail.width = -12
+avail.metric = filesys.avail
+avail.scale = true
```

Comment:
As discussed earlier, let's drop this 'scale' field and make this automatically do the right thing everywhere.

Reply:
I have a branch with this working now, will post an update soon.

Reply:
The generic value scaling work is now merged, marking this one resolved.